web/.env.dev

```diff
@@ -2,3 +2,4 @@ NODE_ENV=development
 __DEV__=true
 #API_URL=https://depviz-demo.moul.io/api
 API_URL=https://depviz-demo-auth.moul.io/api/
+AUTH_TOKEN=d3pviz
```

Comment:
auth token should not be passed as env var but provided by the user with a `window.prompt`

Reply:
@moul in latest commit auth token storing changed. Auth token saved to localstorage and we ask user to enter token in modal form

Reply:
:+1: